SoftLayer/managers/account.py

```diff
@@ -277,7 +277,7 @@ def get_account_all_billing_orders(self, limit=100, mask=None):
         return self.client.call('Billing_Order', 'getAllObjects',
                                 limit=limit, mask=mask)
 
-    def get_routers(self, mask=None, location=None):
+    def get_routers(self, location=None, mask=None):
```

Comment:
I suggest you to fix it direct in the cli hardware create-options module, like this:
```python
routers = account_manager.get_routers(location=location)
```

Reply:
agreed, being specific about parameters can help avoid bugs like this in the future.